crates/node/src/interactive_cli/identity.rs

```diff
@@ -55,21 +56,22 @@ enum AliasSubcommands {
     Remove {
         /// Name of the alias to remove
         identity: Alias<PublicKey>,
-        /// The context that the identity is a member of
-        #[arg(long, short)]
-        context: Alias<ContextId>,
+        /// The context that the identity is a member of (omit to use default context)
```

Comment:
How is this represented in the `--help` report?

Maybe we should break this into a separate line?

Otherwise, we can also omit this, and cover it in the documentation.

Reply:
Did you compare these approaches?